lofo/lofo_importance.py

```diff
@@ -24,9 +24,11 @@ class LOFOImportance:
         Same as cv in sklearn API
     n_jobs: int, optional
         Number of jobs for parallel computation
+    verbose: bool, optional
+        Whether to print progress bar or not
     """
 
-    def __init__(self, dataset, scoring, model=None, fit_params=None, cv=4, n_jobs=None):
+    def __init__(self, dataset, scoring, model=None, fit_params=None, cv=4, n_jobs=None, verbose=False):
```

Comment:
I think default should be True so that people can continue using it as they do.

Reply:
yes, fixed